packages/eas-cli/src/commands/build/submit.ts

```diff
@@ -75,13 +75,31 @@ export default class BuildSubmit extends Command {
     'apple-id': flags.string({
       description: 'Your Apple ID username (you can also set EXPO_APPLE_ID env variable)',
     }),
-    'apple-app-specific-password': flags.string({
-      description:
-        'Your Apple ID app-specific password. You can also set EXPO_APPLE_APP_SPECIFIC_PASSWORD env variable.',
-    }),
     'app-apple-id': flags.string({
       description: 'App Store Connect unique application Apple ID number.',
     }),
+    'apple-team-id': flags.string({
+      description: 'Your Apple Developer Team ID',
+    }),
+    'app-name': flags.string({
+      description:
+        "The name of your app as it will appear on the App Store, this can't be longer than 30 characters (default: expo.name from app.json)",
+    }),
+    'bundle-identifier': flags.string({
+      description: 'Your iOS Bundle Identifier (default: expo.ios.bundleIdentifier from app.json)',
+    }),
```

Comment:
Hmm, I'm not sure if this should be a command param. I'm not saying it's undesirable but rather we don't have a similar param for build commands.

Reply:
Hmm, my current flow assumed:
- Get bundle ID from this param
- Then try to gather it from exp config
- Finally, prompt user

But according to the comment below (about using `getBundleIdentifier()`), the flow can be changed to just gather it from param or config (remove prompt).

Providing bundle ID as param / asking for it is useful only in cases when user wants to submit an external binary and/or, if ever possible, to run `eas submit` from outside project directory (manifest not available).